packages/mui-material/src/Select/SelectInput.js

```diff
@@ -571,6 +602,7 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
             ...listProps,
           },
           paper: {
+            ref: paperRef,
```

Comment:
A ref could be comming in `paperProps` that breaks this, doesn't it? We should probably merge `paperRef`, `paperProps.ref`, and pass that.